From 1146a0bdfed2a8d3f9acd7fe422da1c31bdc4d6d Mon Sep 17 00:00:00 2001
From: "Reddy, Alavala Srinivasa" <alavala.srinivasa.reddy@intel.com>
Date: Mon, 7 Aug 2023 20:19:29 +0530
Subject: [PATCH] Update security_patch_level string

Security_patch_level needs to be updated.
When ASB patches are integrated.

Signed-off-by: Reddy, Alavala Srinivasa <alavala.srinivasa.reddy@intel.com>
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 0daae6bdcb..f7bb875463 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -240,7 +240,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-    PLATFORM_SECURITY_PATCH := 2022-06-05
+    PLATFORM_SECURITY_PATCH := 2023-07-01
 endif
 .KATI_READONLY := PLATFORM_SECURITY_PATCH
 
-- 
2.17.1

